tests/cupyx_tests/scipy_tests/interpolate_tests/test_polyint.py

```diff
@@ -0,0 +1,210 @@
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        D = P.derivatives(test_xs)
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_low_derivatives(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = xp.poly1d([-2, 3, 1, 5, -4])
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        D = P.derivatives(test_xs, len(xs) + 2)
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_derivative(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = xp.poly1d([-2, 3, 1, 5, -4])
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        m = 10
+        D = P.derivative(test_xs, m)
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_high_derivative(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = xp.poly1d([-2, 3, 1, 5, -4])
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        D = P.derivative(test_xs, 2 * len(xs))
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_hermite(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = xp.poly1d([-2, 3, 1, 5, -4])
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        D = P(test_xs)
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_vector(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind == 'u':
+            pytest.skip()
+        xs = xp.array([0, 1, 2], dtype=dtype)
+        ys = xp.array([[0, 1], [1, 0], [2, 1]], dtype=dtype)
+        test_xs = xp.linspace(-1, 3, 5, dtype=dtype)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        return P(test_xs)
+
+    @testing.for_dtypes('bhilfdFD')
```

Comment:
No complex is okay here.
```suggestion
    @testing.for_dtypes('bhilfd')
```

Reply:
updated, thanks!